pkgs/development/compilers/silice/default.nix

```diff
@@ -1,32 +1,38 @@
 { stdenv, fetchFromGitHub, lib
 , cmake, pkg-config, openjdk
 , libuuid, python3
+, glfw
 , silice, yosys, nextpnr, verilator
```

Comment:
```suggestion
, yosys, nextpnr, verilator
```

Reply:
good catch, i originally had it in but lost it while rewriting git history